extensions/amp-story/1.0/amp-story-embedded-component.js

```diff
@@ -954,6 +961,31 @@ export class AmpStoryEmbeddedComponent {
     );
   }
 
+  /**
+   * Show or hide arrows based on current page
+   */
+  updateNavButtons_() {
+    if (!this.isLastPage_()) {
+      this.buttonLeft_.removeAttribute('hidden');
+      this.buttonRight_.removeAttribute('hidden');
+    } else {
+      this.storeService_.get(StateProperty.RTL_STATE)
+        ? this.buttonLeft_.setAttribute('hidden', true)
+        : this.buttonRight_.setAttribute('hidden', true);
+    }
+  }
+
+  /**
+   * Is last page.
+   * @return {boolean}
+   * @private
+   */
+  isLastPage_() {
+    const pageIndex = this.storeService_.get(StateProperty.CURRENT_PAGE_INDEX);
+    const pageCount = this.storeService_.get(StateProperty.PAGE_IDS).length;
+    return pageIndex + 1 >= pageCount;
```

Comment:
nit: change to `pageIndex + 1 === pageCount` or `pageIndex === pageCount - 1` since `pageIndex + 1 > pageCount shouldn't happen`

Reply:
thanks thanks!